csm/conf/uds.py

```diff
@@ -206,7 +206,10 @@ def update_uds_config(cls, uds_public_ip):
 
     @classmethod
     def remove_uds_config(cls):
-        rmtree(cls.UDS_CONFIG_DIR)
+        try:
+            rmtree(cls.UDS_CONFIG_DIR)
+        except FileNotFoundError:
+            pass
```

Comment:
Please Log the error messege here instead of just `pass`.

Reply:
The idea here is to have it work like `rm -f`.  No error message should be expected.

Reply:
`rmtree(cls.UDS_CONFIG_DIR)` is in try..except block and FileNotFound is an expected error... It can be logged in case this error raises.

Reply:
It is not an error.  In the same way `rm -f` does not give you an "Error: File not found" message when you attempt to remove a file that does not exist, there is no need to log such cases.

Please run `rm -f /tmp/non_existing_file` in your shell and verify if an error message is returned. It is not.

Reply:
I agree no error message is required here. Howvere a debug/info statement would be helpful here. Please add loggers

Reply:
Making it a debug log is useless.  The log level for `csm_setup` is hardcoded.

From [csm/conf/csm_setup.py](https://github.com/Seagate/cortx-manager/blob/586fc2d408bb2c4001c342230c3693cb6adf5ad4/csm/conf/csm_setup.py#L37):
```
        Log.init(service_name = "csm_setup", log_path = const.CSM_SETUP_LOG_DIR,
                level=const.LOG_LEVEL)
```
The [value of `const.LOG_LEVEL` on csm/core/blogic/const.py](https://github.com/Seagate/cortx-manager/blob/586fc2d408bb2c4001c342230c3693cb6adf5ad4/csm/core/blogic/const.py#L415) is `"INFO"`.

Using "INFO" level is giving too much importance to a non-event.  I am sure that if UDS doesn't work as expected in mgmt IP mode people will come to me saying something along the lines of "the setup logs are saying that the configuration directory is not there"---and it should not be.  I don't want that.